src/libelektra/kdb.c

```diff
@@ -226,6 +226,16 @@ KDB * kdbOpen(Key *errorKey)
 		errno = errnosave;
 		return handle;
 	}
+	if(elektraMountGlobals(handle, ksDup(keys), handle->modules, errorKey) == -1)
+	{
+#if DEBUG && VERBOSE
+		printf("Mounting global plugins failed\n");
+#endif
+		kdbClose(handle, errorKey);
+		keyDel(initialParent);
+		return 0;
```

Comment:
it's possible that mounting a plugin failes, but it should just yield an error/warning and not crash everything ?